token/core/fabtoken/setup.go

```diff
@@ -8,27 +8,36 @@ package fabtoken
 
 import (
 	"encoding/json"
-	"github.com/hyperledger-labs/fabric-smart-client/platform/view/view"
-	"github.com/pkg/errors"
 
+	"github.com/hyperledger-labs/fabric-smart-client/platform/view/view"
 	"github.com/hyperledger-labs/fabric-token-sdk/token/driver"
+	"github.com/pkg/errors"
 )
 
 const (
-	Coin             = uint64(1000000000)
-	MaxMoney         = uint64(21000000) * Coin
+	Coin = uint64(1000000000)
```

Comment:
What is `Coin`?

Reply:
these parameters are not used. they will be removed.